docs/advanced-usage/connecting-cloud-storage/amazon-s3.md

```diff
@@ -0,0 +1,111 @@
+Integrations can be established to Amazon S3.
+
+To get started, ensure you have admin access within Kolena.
+Navigate to your [Organization Settings's Integrations Tab](https://app.kolena.io/redirect/organization?tab=integrations) and click "Add Integration", then "AWS".
+
+### 1. Allow CORS Access to Bucket
+
+CORS permissions are required for the Kolena domain to render content from your bucket.
+
+Navigate to your S3 bucket inside your AWS console.
+Click on the "Permissions" tab and navigate to the "Cross-origin resource sharing (CORS)" section.
+Click "Edit" and add the following JSON snippet:
+
+```json
+[
+  {
+    "AllowedHeaders": ["*"],
+    "AllowedMethods": ["GET"],
+    "AllowedOrigins": ["https://app.kolena.io"],
+    "ExposeHeaders": []
+  }
+]
+```
+
+### 2. Create an Access Policy in AWS
+
+First ensure you have IAM write permissions within your AWS account.
+In your AWS console, navigate to the IAM policies page.
+Click the "Create Policy" button and select the "JSON" tab.
+
+Paste the following JSON policy (please replace the bucket name `share-with-kolena` with the appropriate bucket)
```

Comment:
> (please replace the bucket name `share-with-kolena` with the appropriate bucket)

How can we make this more obvious? Seems like an easy step to overlook.

Reply:
Making this a note seems to elevate the message appropriately:

![Screenshot 2023-08-22 at 11 26 47 AM](https://github.com/kolenaIO/kolena/assets/20345725/2f9c53ca-4528-47bd-82c1-d69155d6b8fa)